### DISCUSSION TITLE
opencv and pybricks

### DISCUSSION THREAD

**Discussion by DexterTaha** [pybricks/support#1611] [General]
there is any methode that alow me to make relation btw opencv and pybricks code
to cotrole a spike or robot inventor hube for exmple when i show my webcan light trafic the same light be in hub button

**Comment by laurensvalk** [comment]
You could make a script on your computer (with open CV) communicate with the robot using this technique: https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication/
